src/graphics/font.cpp

```diff
@@ -213,9 +213,15 @@ glm::vec2 Font::PreparedFontString::getUsedAreaSize() const
 {
     float max_x = 0.0f;
     float max_y = 0.0f;
+    float line_y = 0.0f;
     for(auto& d : data) {
-        max_x = std::max(max_x, d.position.x);
-        max_y = std::max(max_y, d.position.y);
+        if (d.char_code == 0) {
+            max_y += line_y;
+            line_y = 0.0f;
+        }else{
+            max_x = std::max(max_x, d.position.x);
+            line_y = std::max(line_y, 1.3f * d.size);
```

Comment:
For reference, this is what it looks like with 1.2f

![image](https://github.com/user-attachments/assets/d42fe5f9-0c01-473c-a2e4-6839b649d891)